torch/csrc/api/src/nn/modules/pooling.cpp

```diff
@@ -197,5 +197,29 @@ template class MaxUnpoolImpl<1, MaxUnpool1dImpl>;
 template class MaxUnpoolImpl<2, MaxUnpool2dImpl>;
 template class MaxUnpoolImpl<3, MaxUnpool3dImpl>;
 
+// ============================================================================
+
+template <size_t D, typename Derived>
+LPPoolImpl<D, Derived>::LPPoolImpl(const LPPoolOptions<D>& options_)
+    : options(options_) {}
+
+template <size_t D, typename Derived>
+void LPPoolImpl<D, Derived>::reset() {}
+
+template <size_t D, typename Derived>
+void LPPoolImpl<D, Derived>::pretty_print(std::ostream& stream) const {
+  stream << std::boolalpha
+         << "torch::nn::LPPool" << D << "d("
```

Comment:
It would be awesome to print `norm_type` as well, to match the Python version even better :D

Reply:
Added `norm_type` to print.